v4/platformUI/src/main/java/exchange/dydx/platformui/components/buttons/PlatformButton.kt

```diff
@@ -104,11 +105,10 @@ fun PlatformButton(
                 leadingContent()
             }
             if (text != null) {
-                Text(
-                    style = TextStyle.dydxDefault.themeFont(fontSize = fontSize)
+                PlatformAutoSizingText(
```

Comment:
Target leverage didn't fit when it was 4 digits (ex: 13.58x) which comes up when initializing to existing position leverage